marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
...@pamval, and also to provide single point of usage httpclient in pentaho products